### DISCUSSION TITLE
3rd person incompatible

### DISCUSSION THREAD

**Discussion by SETKA-SIMULATORS** [stijnwop/manureSystem#173] [General]
Hello, why is the 3rd person mod incompatible with the Manure System mod? If I have it activated in the game, the hoses do not connect. Sasludos

**Comment by stijnwop** [comment]
Please contact the modder of 3rd person.
See:
https://github.com/stijnwop/manureSystem?tab=readme-ov-file#known-conflicts

**Comment by Tutu3862** [comment]
Hello, I have the hose for the herbicide and the Rubicon 9000.
I don't know where to connect it or how to connect it to the herbicide tank.
